### PR TITLE
Source code for "Precision and OMP SIMD article"

### PR DESCRIPTION
**Pull request by ibogosavljevic**

### CODE REVIEW

**Comment by ibogosavljevic** [comment]
A gentle reminder

File: precision-omp-simd/compile.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+set -o xtrace
+
+ARGS="-g -O3 -march=core-avx2 -fno-math-errno -Rpass-missed=loop-vectorize -Rpass=loop-vectorize -Rpass-analysis=loop-vectorize"
+ARGS="-g -O3 -march=core-avx2 -fno-math-errno"
```
Comment (javinovo) [inline precision-omp-simd/compile.sh:4]:
```suggestion
```
Same as the PR#4, this is for `gcc` and can be removed, right?

File: precision-omp-simd/compile.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+CC=clang
+${CC} ${ARGS} precision-omp-simd.c -o precision-noomp-simd  -lm
+${CC} ${ARGS} -fopenmp precision-omp-simd.c -o precision-omp-simd -lm
+
+echo "Without -fopenmp"
+./precision-noomp-simd
+
+echo "With -fopenmo"
```
Comment (javinovo) [inline precision-omp-simd/compile.sh:12]:
```suggestion
echo "With OpenMP vectorization"
```

File: precision-omp-simd/compile.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+set -o xtrace
+
+ARGS="-g -O3 -march=core-avx2 -fno-math-errno -Rpass-missed=loop-vectorize -Rpass=loop-vectorize -Rpass-analysis=loop-vectorize"
+ARGS="-g -O3 -march=core-avx2 -fno-math-errno"
+CC=clang
+${CC} ${ARGS} precision-omp-simd.c -o precision-noomp-simd  -lm
+${CC} ${ARGS} -fopenmp precision-omp-simd.c -o precision-omp-simd -lm
+
+echo "Without -fopenmp"
```
Comment (javinovo) [inline precision-omp-simd/compile.sh:9]:
```suggestion
echo "Without OpenMP vectorization"
```

File: precision-omp-simd/precision-omp-simd.c [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    printf("%f, 5 / a = %f\n", a, 5 / a);
+    show_fe_exceptions();
+    clear_exceptions();
+
+    printf("sqrt(%f) = %f\n", a, sqrtf(a));
+    show_fe_exceptions();
+    clear_exceptions();
+
+    return;
+}
```
Comment (javinovo) [inline precision-omp-simd/precision-omp-simd.c:43]:
```suggestion
```

Same as PR#5, this is unused code

**Review by javinovo — CHANGES_REQUESTED** [review]

File: precision-omp-simd/compile.sh (hunk 1 above)
Comment (ibogosavljevic) [reply precision-omp-simd/compile.sh:4]: You mean to remove the other line, right??

Comment (javinovo) [reply precision-omp-simd/compile.sh:3]: No, the one with the Clang vectorization messages should stay, don't you think? The messages are quite important!

**Comment by ibogosavljevic** [comment]
Ok, let me fix it then.

**Pull request merged**